src/javascript/EditPanel/WorkInProgress/OpenWorkInProgressModal.action.jsx

```diff
@@ -0,0 +1,50 @@
+
+import React, {useContext} from 'react';
+import PropTypes from 'prop-types';
+import WorkInProgressDialog from './WorkInProgressDialog/WorkInProgressDialog';
+import {ComponentRendererContext} from '@jahia/ui-extender';
+
+export const OpenWorkInProgressModal = ({context, render: Render}) => {
+    const componentRenderer = useContext(ComponentRendererContext);
+
+    const {siteInfo} = context;
+    const closeDialog = () => {
+        componentRenderer.destroy('WorkInProgressDialog');
+    };
+
+    return (
+        <>
+            <Render context={{
```

Comment:
Avoid passing everything in the context object. Use plain React props ;)

Reply:
I don't  agree on that one. the context contain all props that will be used to render the Render component properly, if we don't pass them all, we might miss some (as an action my have no knowledge of its usage)